src/containers/Main/Vote/index.tsx

```diff
@@ -11,16 +11,11 @@ import Proposals from 'components/Vote/Proposals';
 import { promisify } from 'utilities';
 import { Row, Column } from 'components/Basic/Style';
 import { useWeb3React } from '@web3-react/core';
-import useRefresh from '../../hooks/useRefresh';
-import { CONTRACT_XVS_TOKEN_ADDRESS, CONTRACT_VBEP_ADDRESS } from '../../utilities/constants';
-import {
```

Comment:
I wouldn't expect this to change I think we still have conflicts between eslint and prettier